src/agent/src/storage/fs_handler.rs

```diff
@@ -50,6 +51,14 @@ impl StorageHandler for OverlayfsHandler {
                 .options
                 .push(format!("workdir={}", work.to_string_lossy()));
         }
+        for driver_option in &storage.driver_options {
```

Comment:
```
        let prefix = format!("{}=", KATA_VOLUME_OVERLAYFS_CREATE_DIR;
        for driver_option in &storage.driver_options {
            if let Some(dir) = driver_option.as_str().strip_prefix(&prefix)) {
```
It would be great to use https://docs.rs/safe-path/0.1.0/safe_path/struct.ScopedDirBuilder.html to create directory.

Reply:
Maybe we can create a new pr to enhance the directory creation.